src/main/java/br/com/maykmoreira/apiexample/service/MovieService.java

```diff
@@ -0,0 +1,13 @@
+package br.com.maykmoreira.apiexample.service;
+
+import br.com.maykmoreira.apiexample.model.Movie;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+
+public interface MovieService {
```

Comment:
There are many ways we can name an interface. Here we are used to naming it starting with "I" letter. So, in this case, your interface should be called IMovieService

Reply:
Note that I'm not saying this is the better way, it's just the way we use it here =)